main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/appc/docker2aci/lib"
+	"github.com/coreos/rocket/cas"
+)
+
+const (
+	rocketDir = "/var/lib/rkt"
+)
+
+var flagImport = flag.Bool("import", false, "Import ACI images to the rocket store")
```

Comment:
You beat me to it :-). This absolutely should not be here, there will be no Rocket related code in appc. It was just from early prototyping.

What's here now will end up in Rocket itself - tracking ticket: https://github.com/coreos/rocket/issues/145